docs/release_process.md

```diff
@@ -131,7 +127,7 @@ Using `release_helper.py` script:
 ### Bumping the Version
 
 - Open a PR to bump the version of `CMakeLists.txt` on the **master** branch.
-    - For example, when the `stabilization-v0.1.65` is created, the line `set(SSG_PATCH_VERSION 65)`
+    - For example, if the version  `0.1.65` is going to be released, the line `set(SSG_PATCH_VERSION 65)`
```

Comment:
I think the proper phrasing is "if the version ... is to be released" rather than using "going to be" (future tense in a condition).

Reply:
Interesting catch, but it seems that even the current phrasing is fine.
https://ell.stackexchange.com/questions/177544/be-to-verb-vs-be-going-to-verb

Reply:
I am still not convinced, my problem was with the conditional, see ie. https://www.ef.com/wwen/english-resources/english-grammar/conditional/ .

Basically, the problem is the "if future tense, then ..." - it's a typical "czechism" because slavic languages commonly use future tense in conditionals, but it's fairly uncommon / incorrect (?) in English where the default tends to be a time-neutral condition "if some thing happens, then ..." with other (past) forms allowed, but less common.